docs/features/authentication.md

```diff
@@ -1,17 +1,39 @@
 # Authentication
-Authentication is an enchanced Identity feature of mxisd to ensure coherent and centralized identity management.
+
+- [Description](#description)
+- [Overview](#overview)
+- [Getting started](#getting-started)
+  - [Synapse](#synapse)
+  - [mxisd](#mxisd)
+  - [Validate](#validate)
+- [Next steps](#next-steps)
+  - [Profile auto-fil](#profile-auto-fill)
+- [Advanced Authentication](#advanced-authentication)
+  - [Requirements](#requirements)
+  - [Configuration](#configuration)
+    - [Reverse Proxy](#reverse-proxy)
+      - [Apache2](#apache2)
+    - [DNS Overwrite](#dns-overwrite)
+    - [Backends](#backends) 
+
+## Description
+Authentication is an enhanced Identity feature of mxisd to ensure coherent and centralized identity management.
```

Comment:
- Not necessary centralized - you can have several identity stores.
- `Identity management` might be misleading, as mxisd doesn't do any management, only reads data.

I like the spirit of the sentence, but not the wording :D I also don't have a good idea about the wording at this point